packages/flutter_test/lib/src/binding.dart

```diff
@@ -544,6 +544,12 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     );
   }
 
+  /// Expose this method for testing.
```

Comment:
This exposes the method for every test (not only ours, but also for all our customers). Since there should never be a reason for anyone to call this, we shouldn't do that. You'll have to create a custom binding subclass that's only used for your test.

Reply:
As only one binding can be active at a time, to my knowledge, I've put this test into a new file so that the custom binding it uses can be used exclusively by this test.